### PR TITLE
Add lab2

### PR DESCRIPTION
**Pull request by akshatp11**
@mrdarcychen

### CODE REVIEW

File: Lab2.ipynb [hunk 1]
```diff
@@ -0,0 +1,761 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If we consider N to be 7 days that have passed, over these days we see N*Y = 56 gamma rays. In order to determine the sigma of this observation we need to first describe the question that we are asking. What is the probability that the cosmic ray background data gives us something resembling 56 or more gamma rays? The probability to for this to occur would be the integral from 56 to infinty of the probability distribution of the background. This can also be done by subtracting the integral of whatever is before 56 from 1 which is the method used below. "
```
Comment (mrdarcychen) [inline Lab2.ipynb]: typo "infinty"

File: Lab2.ipynb [hunk 2]
```diff
@@ -0,0 +1,761 @@
...
+   "source": [
+    "sigma = stats.norm.ppf(1 - prob)\n",
+    "print('The sigma corresponding to this probability is ' + str(round(sigma, 2)) + 'σ')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This sigma value is significant since in physics sigma values over 5σ tend to be considered signifiacnt."
```
Comment (mrdarcychen) [inline Lab2.ipynb]: typo "signifiacnt"

**Comment by mrdarcychen** [comment]
Hi Akshat, I'm a bit confused on what day zero means in In[15] of problem 1 because the inner for loop doesn't get executed with range(0). I've also pointed out a few typos in your report. Otherwise everything looks great!

**Comment by akshatp11** [comment]
That array refers to a range of days so the zero is just the original one since no days have passed.

**Comment by akshatp11** [comment]
@gracesong312

**Pull request merged**